northstar/src/npk/manifest/console.rs

```diff
@@ -5,10 +5,22 @@ use serde::{
     ser::SerializeSeq,
     Deserialize, Serialize, Serializer,
 };
+use serde_with::skip_serializing_none;
 use std::{collections::HashSet, fmt};
 use strum::{EnumCount, IntoEnumIterator};
 use strum_macros::{EnumCount, EnumIter};
 
+/// Console Quality of Service
+#[skip_serializing_none]
+#[derive(Clone, PartialEq, Eq, Debug, Default, Serialize, Deserialize, JsonSchema)]
+#[serde(deny_unknown_fields)]
+pub struct ConsoleQoS {
```

Comment:
What's wrong by naming the struct `Console`? It's everything console related for this container...

Reply:
I think that `Console` is too generic for this group of options and there is already a `Console` struct that handles the console service. Is `ConsoleConfiguration` okay? `ConsoleOps`?